x-pack/plugins/ml/public/application/util/get_selected_ids_url.ts

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Could you please implement it as a method in ML URL generator?

Reply:
There is an existing issue for consolidating the url generating functions for 7.10 that @qn895 is working on so this will be addressed by that afaik. 
https://github.com/elastic/kibana/issues/69265#issuecomment-659534242